kind/actions/cluster_actions.go

```diff
@@ -306,8 +306,8 @@ func GetNodeRefUID(clusterName, nodeName string) (string, error) {
 	return strings.TrimSpace(lines[0]), nil
 }
 
-// DeleteClusterNode will remove the kubernetes node from the list of nodes (during a kubectl get nodes).
-func DeleteClusterNode(clusterName, nodeName string) error {
+// DeleteNode will remove the kubernetes node from the list of nodes (during a kubectl get nodes).
```

Comment:
I have a PR out that fixes this issue, this isn't quite right since the delete action became two different things to work properly. DeleteClusterNode is the thing that both DeleteWorker and DeleteControlPlane call after they do the steps necessary for deleting a worker and deleting a control plane

Reply:
without the delete rename, the build fails for me.

Reply:
I have a PR out that uses the correct methods in kind-test.

Reply:
https://github.com/kubernetes-sigs/cluster-api-provider-docker/pull/14

Reply:
ok, found https://github.com/kubernetes-sigs/cluster-api-provider-docker/pull/14